### DISCUSSION TITLE
[A-L] (8 Febbraio 2022) Esercizio 3

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#300] [Esami A-L]
![image](https://user-images.githubusercontent.com/69389461/153033240-7e9e71b4-e420-4622-bf41-2c30379428f7.png)

**Comment by matypist** [comment]
1. Transitiva? [Se R(a, b) e R(b, c) allora R(a, c)?] V
2. Antisimmetrica? [Se R(a, b) e R(b, a) allora a = b?] V
3. Totalità? [Per ogni a, b ∈ A vale a ≤ b o b ≤ a?] V

**Comment by CiottoloMaggico** [comment]
![immagine](https://user-images.githubusercontent.com/79638284/216340100-ed199a49-7006-4a45-81f1-a1b662b0f951.png)
![immagine](https://user-images.githubusercontent.com/79638284/210134332-3fdc145d-b7ff-4788-95c2-ad3b05bf4216.png)
![immagine](https://user-images.githubusercontent.com/79638284/210134325-cd9e14f0-8298-477f-89da-97961dcbddee.png)

**Comment by aflaag** [comment]
@CiottoloMaggico l'antisimmmetria invece è sempre verificata perché una funzione si identifica con i punti che assume, indipendentemente dalla notazione insiemistica se due funzioni, anche analiticamente, coincidono è perché assumono gli stessi valori in certi punti, quindi anche avendo as esempio $S= \\{0 \\}$ con $f(x)= \sin(x)$ e $g(x)=x$, l'antisimmmetria è verificata perche $x=0 \implies f(x)=g(x)=0$

**Comment by Exyss** [comment]
## Dimostrazione analoga a #84

**Comment by CiottoloMaggico** [comment]
@ph04 Con l'appunto volevo intendere proprio questo, se due funzioni con regole diverse generano lo stesso "insieme di definizione" ovvero l'insieme di tutte le coppie (x, y) vengono cosiderate la stessa funzione.....
Di conseguenza come hai detto tu un esempio è f(x) = sin(x) e g(x) = x quando S = {0}.
Comunque mi sono dimenticato di postare la dimostrazione della riflessività anche se mi sembra abbastanza banale, per il resto 
riflessività: Vero
anti-simmetria: Vero
transitività: Vero
Totale: Falso

La relazione non è un ordine totale poichè la totalità non è rispettata.
